src/quality_result_gui/quality_error_manager.py

```diff
@@ -157,6 +162,41 @@ def _add_predefined_filters(self) -> None:
             self._attribute_filter.update_filter_from_errors
         )
 
+    def _add_actions(self) -> None:
```

Comment:
Would it work if we just create a simple QShortcut and connect that to checbox's toggle method? Something like:
```python
    def _add_shortcuts(self) -> None:
        self.shortcut = QShortcut(QualityResultManager.SHORTCUT_TOGGLE_ERRORS_ON_MAP_FILTER)
        self.shortcut.setObjectName(tr("Toggle show errors on map filter"))
        self.shortcut.activated.connect(self.dock_widget.show_errors_on_map_check_box.toggle)

        QgsGui.shortcutsManager().registerShortcut(self.shortcut)
```

Reply:
Implemented as suggested, action removed